Apsitvarkom.UnitTests/DataAccess/PollutedLocationDTODatabaseRepositoryTests.cs

```diff
@@ -0,0 +1,194 @@
+﻿using Apsitvarkom.DataAccess;
+using Apsitvarkom.Models;
+using AutoMapper;
+using Apsitvarkom.Models.Mapping;
+using Moq;
+using MockQueryable.Moq;
+using System.Globalization;
+using static Apsitvarkom.Models.Enumerations;
+
+namespace Apsitvarkom.UnitTests.DataAccess;
+
+public class PollutedLocationDTODatabaseRepositoryTests
+{
+    private IMapper m_mapper = null!;
+    private Mock<IPollutedLocationContext> m_mockContext = null!;
+
+    [SetUp]
+    public void SetUp()
+    {
+        var config = new MapperConfiguration(cfg =>
+        {
+            cfg.AddProfile<PollutedLocationProfile>();
+        });
+        config.AssertConfigurationIsValid();
+        m_mapper = config.CreateMapper(); 
+        m_mockContext = new Mock<IPollutedLocationContext>();
+    }
```

Comment:
Would be possible to extract some things into `[OneTimeSetUp]`, right? `m_mapper` does not change/mutate between tests